clients/src/test/java/org/apache/kafka/common/requests/ProduceRequestTest.java

```diff
@@ -164,7 +188,7 @@ public void testV3AndAboveCannotUseMagicV0() {
         ProduceRequest.Builder requestBuilder = ProduceRequest.builder(new ProduceRequestData()
             .setTopicData(new ProduceRequestData.TopicProduceDataCollection(Collections.singletonList(
                 new ProduceRequestData.TopicProduceData()
-                    .setName("test")
+                    .setTopicId(Uuid.fromString("H3Emm3vW7AKKO4NTRPaCWt"))
```

Comment:
There are still quite a few other usage of `setName`. Should we change them to use `setTopicId`?